apps/server/src/modules/files-storage/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from './files-storage-amqp.module';
+export * from './files-storage-api.module';
+export * from './files-storage-test.module';
```

Comment:
Can you put a comment to this line that is come deprecated if the test are moved?